tests/rptest/scale_tests/many_clients_test.py

```diff
@@ -55,7 +56,8 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
     @cluster(num_nodes=7)
-    def test_many_clients(self):
+    @matrix(idempotent_producers=[True, False])
```

Comment:
usually impotency is on by default, are you sure it was set to off for ProducerSwarm?

Reply:
yes, client_swarm uses Rust client based on `librdkafka`

Reply:
IIUC correct the goal here is just to verify that it works in both cases?

Reply:
exactly